providers/dell/idrac8/configure.go

```diff
@@ -281,26 +279,23 @@ func (i *IDrac8) applyNtpServerParam(cfg *cfgresources.Ntp) {
 		enable = 1
 	}
 
-	//https://10.193.251.10/data?set=tm_ntp_int_opmode:1, \\
-	//                               tm_ntp_str_server1:ntp0.lhr4.example.com, \\
-	//                               tm_ntp_str_server2:ntp0.ams4.example.com, \\
+	// https://10.193.251.10/data?set=tm_ntp_int_opmode:1,
+	//                               tm_ntp_str_server1:ntp0.lhr4.example.com,
+	//                               tm_ntp_str_server2:ntp0.ams4.example.com,
 	//                               tm_ntp_str_server3:ntp0.fra4.example.com
 	queryStr := fmt.Sprintf("set=tm_ntp_int_opmode:%d,", enable)
 	queryStr += fmt.Sprintf("tm_ntp_str_server1:%s,", cfg.Server1)
 	queryStr += fmt.Sprintf("tm_ntp_str_server2:%s,", cfg.Server2)
 	queryStr += fmt.Sprintf("tm_ntp_str_server3:%s,", cfg.Server3)
 
-	//GET - params as query string
-	//ntp servers
-
 	endpoint := fmt.Sprintf("data?%s", queryStr)
 	statusCode, response, err := i.get(endpoint, nil)
 	if err != nil || statusCode != 200 {
 		i.log.V(1).Info("GET request failed.",
 			"IP", i.ip,
 			"HardwareType", i.HardwareType(),
 			"endpoint", endpoint,
-			"status", statusCode,
+			"StatusCode", statusCode,
```

Comment:
I've already changed relevant dashboards... the point is that other entries are already capital, and I would do the effort and change all of them but this would make this MR even larger...

Reply:
Agreed to keep it